scout/commands/export/case.py

```diff
@@ -7,6 +7,7 @@
 
 from scout.constants import CASE_STATUSES
 from scout.server.extensions import store
+from scout.server.utils import jsonconverter
 
 from .export_handler import bson_handler
```

Comment:
I think Vulture is right here, it's not used any more

Reply:
Yeah, that likely was the last place.